src/common/test-helpers/button.helper.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { Button } from "@map-colonies/react-core";
+import { ReactWrapper, ShallowWrapper } from "enzyme";
+
+/* eslint-disable */
+export const getButtonById = (wrapper: ReactWrapper | ShallowWrapper, id: string): ReactWrapper | ShallowWrapper => {
+  return wrapper
+    // @ts-ignore
+    .findWhere((n) => {
+      return n.type() === Button &&
+        n.prop('children').props['id'] === id;
+    });
+};
+
+describe.skip('Button helpers dummy suite', () => {
+  it('dummy test', () => {
+    expect(true).toBe(true);
+  });
+});
+/* eslint-enable */
```

Comment:
end of file

Reply:
Done, think it's already irrelevant to check last line